### PR TITLE
Api documentation

### PR DESCRIPTION
**Pull request by MageziArthur**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/resources/static/api-docs/velocity-doc.md [hunk 1]
```diff
@@ -0,0 +1,900 @@
+## API documentation.
+
+### **Authentication**
+Fineract APIs are secured and depending on the security options available in fineract, any api request would require some form of authentication and authorization. Checkout this documentation for more information about available authentication protocols. 
+
+[<u>https://devcredits.velocitycareerlabs.io/fineract-provider/api-docs/apiLive.htm#authentication_overview</u>](https://devcredits.velocitycareerlabs.io/fineract-provider/api-docs/apiLive.htm#authentication_overview)  
+NOTE: Staging instance currently uses Basic Authentication and as such base 64 encoded username and password passed in the Authorization headers of any http request should guarantee access/authorization.
```
Comment (ItayPodhajcer) [inline fineract-provider/src/main/resources/static/api-docs/velocity-doc.md:7]: It's the `Dev` instance, not `Staging`

Comment (ItayPodhajcer) [inline fineract-provider/src/main/resources/static/api-docs/velocity-doc.md:7]: Wasn't the integration with `auth0` already done?

**Review by ItayPodhajcer — CHANGES_REQUESTED** [review]

File: fineract-provider/src/main/resources/static/api-docs/velocity-doc.md (hunk 1 above)
Comment (MageziArthur) [reply fineract-provider/src/main/resources/static/api-docs/velocity-doc.md:7]: This documentation doesn't necessarily cover that and no, auth0 is not yet done

Comment (MageziArthur) [reply fineract-provider/src/main/resources/static/api-docs/velocity-doc.md:7]: Sure, let me change this

**Review by ItayPodhajcer — APPROVED** [review]

**Pull request merged**